### DISCUSSION TITLE
Welcome to open-amp Discussions!

### DISCUSSION THREAD

**Discussion by edmooring** [OpenAMP/open-amp#253] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vishnu-banavath** [comment]
Hi,
My name is Vishnu. I work on ARM based devices, software models and boards. As part of current work, we use openamp stack for communication between Linux and baremetal code.  I'm very much interested in taking part in discussions and understanding the roadmap and any forthcoming design changes.

Thanks
